### DISCUSSION TITLE
Insights / Forks - list does not allow sorting or filtering to see most active/recent/developed forks

### DISCUSSION THREAD

**Discussion by slrslr** [community/community#34166] [Other Feature Feedback, Questions, & Ideas]
The title says it. I wish i can quickly access forks that are most ahead, most recently updated.
Current list is very plain and does not allow that. Example: https://github.com/alidbg/htaccess_firewall/network/members

**Comment by claudio-salvio** [comment]
Hello GitHub,

I totally agree with @slrslr 
The information displayed at **https://github.com/*/*/network/members** is insufficient.
As a consequence it is very difficult to make informed decisions about it.

There are many interesting and useful projects that for various reasons are at a standstill or have been directly abandoned.
It is very common that other developers take over and continue with the development of these projects based on forks.
However, nowadays it is very difficult to detect which of these forks have flourished and have an active and reputable community.
Therefore, following @slrslr 's proposal, I suggest that the **https://github.com/*/*/network/members** pages include information that facilitates the analysis of the forking situation of a project.

For my part the data that I consider relevant for each fork are:
- number of watches
- number of forks
- number of stars
- number of issues
- number of pull requests
- last commit date
- date of last release 

I think it would be very useful if you take into account these suggestions and add information that allows us to act with more information and less effort.

Best regards,
Claudio Salvio
